src/utils/hooks/useVideo/useVideo.test.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import useVideo from './useVideo';
+
+const ComponentTest = () => {
+  const { videos, error } = useVideo('wizeline');
+  return (
+    <div>
+      {error && <div>error</div>}
+      <h1>{videos?.items.count}</h1>
+    </div>
+  );
+};
+
+describe('Video Card component', () => {
+  it('should contains a result', async () => {
```

Comment:
lo correcto sería `should contain`, en este caso no afecta en nada a la funcionalidad, pero creo que no estaba de más mencionarlo 😅